lib/utils/schema/schema.rb

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env ruby
```

Comment:
For safety/paranoia purposes, should we wrap these methods in a module? I'd hate for something to accidentally `require` this and have some methods overridden.